packages/core/src/card/card.stories.ts

```diff
@@ -0,0 +1,147 @@
+    Suspendisse non interdum sem, nec egestas nulla.
+    Duis vitae leo justo. Fusce ante enim, tincidunt nec faucibus sed, fringilla et ipsum.
+    Proin quis diam elit. Cras nec finibus libero. Sed bibendum lectus eget magna ultricies eleifend.
+    Duis ut molestie urna. Proin cursus dolor ac ex rutrum, eu condimentum elit tincidunt.
+    In a dolor faucibus, vehicula ipsum eu, viverra lorem. Aenean congue fermentum commodo.
+    Sed ex purus, consectetur mollis urna eget, vestibulum mollis dolor.
+    Donec sodales, felis sit amet gravida ornare, magna dolor ullamcorper turpis, vel facilisis odio ipsum semper sem.
+  `;
+
+export default {
+  title: 'Stories/Card',
+  component: 'cds-card',
+  argTypes: getElementStorybookArgTypes('cds-card', customElements),
+  parameters: {
+    options: { showPanel: true },
+  },
+};
+
+export function Basic(args: { [key: string]: unknown }) {
+  return html`<cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <p cds-text="body">${placeholder}</p>
+  </cds-card>`;
+}
+
+export function WithLayout() {
+  return html` <cds-card>
+    <div cds-layout="vertical gap:md">
+      <div>
+        <p cds-text="section">Card Title</p>
+      </div>
+
+      <div>
+        <p cds-text="body">${basicContent}</p>
+      </div>
+
+      <cds-divider card-remove-margin></cds-divider>
+
+      <div></div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutAndTwoDividers(args: { [key: string]: unknown }) {
+  return html` <cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <div cds-layout="vertical gap:md">
+      <div>
+        <p cds-text="section">Card Title</p>
+      </div>
+
+      <cds-divider card-remove-margin></cds-divider>
+
+      <div>
+        <p cds-text="body">${basicContent}</p>
+      </div>
+
+      <cds-divider card-remove-margin></cds-divider>
+
+      <div></div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutTwoDividersAndView(args: { [key: string]: unknown }) {
+  return html` <cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <div cds-layout="vertical gap:md">
+      <div>
+        <p cds-text="section">Card Title</p>
+      </div>
+
+      <cds-divider card-remove-margin></cds-divider>
+
+      <div>
+        <p cds-text="body">${basicContent}</p>
+      </div>
+
+      <cds-divider card-remove-margin></cds-divider>
+
+      <div>
+        <cds-button action="flat">View</cds-button>
+      </div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutAndOverflow() {
+  return html` <style>
+      cds-card#cds-card-with-fixed-height #cds-card-with-fixed-height__body {
+        height: 1rem;
+        overflow: auto;
+      }
+    </style>
+    <cds-card id="cds-card-with-fixed-height">
+      <div cds-layout="vertical gap:md">
+        <div>
+          <p cds-text="section">Card Title</p>
+        </div>
+
+        <cds-divider card-remove-margin></cds-divider>
```

Comment:
We should probably prefix this attribute with `cds-` to to ensure its within a scope of sorts and lowers attr collision risk on elements that are not Core.

```html
cds-card-remove-margin
```